lib/was_crawl_preassembly/utilities.rb

```diff
@@ -2,9 +2,8 @@ module Dor
   module WASCrawl
     class Utilities
       def self.get_collection_id(druid_obj)
-        collections = druid_obj.collections
-        raise "#{druid_obj.id} doesn't belong to a collection" if collections.length == 0
-        collection = collections[0]
+        collection = druid_obj.collections.first
```

Comment:
my only question is if `druid.collections.first` will ever raise an error ... and if `druid.collections` itself would.  You test for the first below ... is the second even possible?